api/security/csrf/csrf.py

```diff
@@ -39,6 +41,8 @@ def is_csrf_enabled():
 def set_csrf_cookie(resp, csrf_token):
     resp.set_cookie(CSRF_COOKIE_NAME, value=csrf_token, httponly=True, secure=True, samesite="Lax")
 
+def remove_csrf_cookie(resp):
+    resp.set_cookie(CSRF_COOKIE_NAME, value='', expires=0)
```

Comment:
question: is this better than [delete_cookie](https://flask.palletsprojects.com/en/2.2.x/api/#flask.Response.delete_cookie)?

Reply:
It's not better, I didn't know the `delete_cookie` function. I'll switch to that, nice!